src/parser/mod.rs

```diff
@@ -1816,6 +1829,50 @@ impl<'a, 'o, 'c: 'o> Parser<'a, 'o, 'c> {
             *container = details;
 
             true
+        } else if node_matches!(last_child, NodeValue::DescriptionItem(..)) {
+            // ORIGINAL CODE
+            let parent = last_child.parent().unwrap();
+            reopen_ast_nodes(parent);
+
+            let metadata = NodeDescriptionItem {
+                marker_offset: self.indent,
+                padding: 2,
+                tight,
+            };
+
+            let item = self.add_child(
+                parent,
+                NodeValue::DescriptionItem(metadata),
+                self.first_nonspace + 1,
+            );
+
+            let details =
+                self.add_child(item, NodeValue::DescriptionDetails, self.first_nonspace + 1);
+
+            *container = details;
+
+            true
+
+            // ATTEMPT 1
+            // reopen_ast_nodes(last_child);
+            // 
+            // let details =
+            //     self.add_child(last_child, NodeValue::DescriptionDetails, self.first_nonspace + 1);
+            // *container = details;
+            // 
+            // true
+            
+            // ATTEMPT 2
+            // let parent = last_child.parent().unwrap();
+            // let item = parent.last_child().unwrap();
+            // 
+            // reopen_ast_nodes(item);
+            // 
+            // let details =
+            //     self.add_child(item, NodeValue::DescriptionDetails, self.first_nonspace + 1);
+            // *container = details;
+            // 
+            // true
```

Comment:
@kivikakk I was wondering if you could take a look at this.

I realized that when supporting multiple definitions, the second `DescriptionDetails` gets added as an entirely new `DescriptionItem`, rather than being added as a child to the existing `DescriptionItem`.

But both my attempts to do this resulted in the `assert!(ast.open);` in `finalize_borrowed` tripping.

Sample input

```
foo
: bar
: one
```

I thought maybe it's the shenanigans I'm doing at line 1760, the `None` case, but using the following input (with blanks lines) bypasses that and it still asserts.

```
foo

: bar

: one
```

Even with re-opening the ast node, it's not working. I'm missing some magic incantation. 🤔

Reply:
I couldn't get multiple detail nodes added to a description item. But thinking about it again, I really think it's ok - it just means that there can be multiple `DescriptionItem` with just a `DescriptionDetails`, but no term. Actually I think this is fine. The HTML formatter doesn't care.